internal/errcode/code.go

```diff
@@ -126,6 +126,13 @@ func IsAccountSuspended(err error) bool {
 	return errors.As(err, &e) && e.AccountSuspended()
 }
 
+// IsUnavailableForLegalReasons will check if err or one of its causes was due to
+// legal reasons.
+func IsUnavailableForLegalReasons(err error) bool {
+	var e interface{ UnavailableForLegalReasons() bool }
```

Comment:
Does this work? Doesn't it need to be

```suggestion
	var e interface{ IsUnavailableForLegalReason() bool }
```
?

Reply:
..or renamed below :)